ui/pages/notification-details/notification-details.stories.tsx

```diff
@@ -0,0 +1,201 @@
+  createMockNotificationLidoStakeCompleted,
+  createMockNotificationLidoWithdrawalCompleted,
+  createMockNotificationLidoWithdrawalRequested,
+  createMockNotificationMetaMaskSwapsCompleted,
+  createMockNotificationRocketPoolStakeCompleted,
+  createMockNotificationRocketPoolUnStakeCompleted,
+  createMockFeatureAnnouncementRaw,
+} = NotificationServicesController.Mocks;
+
+// Mock data
+const mockNotifications = {
+  ethSent: createMockNotificationEthSent,
+  ethReceived: createMockNotificationEthReceived,
+  erc20Sent: createMockNotificationERC20Sent,
+  erc20Received: createMockNotificationERC20Received,
+  erc721Sent: createMockNotificationERC721Sent,
+  erc721Received: createMockNotificationERC721Received,
+  erc1155Sent: createMockNotificationERC1155Sent,
+  erc1155Received: createMockNotificationERC1155Received,
+  lidoReadyToBeWithdrawn: createMockNotificationLidoReadyToBeWithdrawn,
+  lidoStakeCompleted: createMockNotificationLidoStakeCompleted,
+  lidoWithdrawalCompleted: createMockNotificationLidoWithdrawalCompleted,
+  lidoWithdrawalRequested: createMockNotificationLidoWithdrawalRequested,
+  metaMaskSwapsCompleted: createMockNotificationMetaMaskSwapsCompleted,
+  rocketPoolStakeCompleted: createMockNotificationRocketPoolStakeCompleted,
+  rocketPoolUnStakeCompleted: createMockNotificationRocketPoolUnStakeCompleted,
+  featureAnnouncement: createMockFeatureAnnouncementRaw,
+};
+
+const processedNotifications = Object.fromEntries(
+  Object.entries(mockNotifications).map(([key, createMock]) => [
+    key,
+    processNotification(createMock()),
+  ]),
+);
+
+const store = configureStore({
+  ...testData,
+});
+
+export default {
+  title: 'Pages/Notifications/NotificationDetails',
+  component: NotificationDetails,
+  decorators: [
+    (Story) => (
+      <Provider store={store}>
+        <Story />
+      </Provider>
+    ),
+  ],
+} as Meta;
+
+const Template = ({ notification }) => {
+  const ncs = NotificationComponents[notification.type];
+
+  return (
+    <Box marginLeft={'auto'} marginRight={'auto'}>
+      <NotificationsPage>
+        <NotificationDetailsHeader
+          onClickBack={() => console.log('click back')}
+        >
+          <ncs.details.title notification={notification} />
+        </NotificationDetailsHeader>
+        <Content padding={0}>
+          <Box
+            display={Display.Flex}
+            flexDirection={FlexDirection.Column}
+            gap={2}
+            width={BlockSize.Full}
+            height={BlockSize.Full}
+            justifyContent={JustifyContent.spaceBetween}
+          >
+            <NotificationDetailsBody
+              body={ncs.details.body}
+              notification={notification}
+            />
+            <NotificationDetailsFooter
+              footer={ncs.footer}
+              notification={notification}
+            />
+          </Box>
+        </Content>
+      </NotificationsPage>
+    </Box>
+  );
+};
+
+export const EthSent = Template.bind({});
```

Comment:
Nice cleanup, lets either add this now or in the next subsequent PR.

For context, the number of notification types we will support is going to grow substantially over time. So being able to create storybooks for each one easily (without much effort) will be important.
- Web 3 Notifications: `aave_v3_health_factor`; `ens_expiration`; `lido_staking_rewards`; `notional_loan_expiration`; `rocketpool_staking_rewards`; `spark_fi_health_factor`
- Price Alerts: Native Token Alerts, ERC Alerts (token/nft alerts)
- And more (security, etc)

Reply:
Thanks, Andrea! Actually, I was thinking of optimizing these stories in a new PR I’m working on because we’re introducing new stories (as Marco mentioned). But you’re right; better to do it right away!